client/src/components/squids/SquidList.jsx

```diff
@@ -12,32 +13,42 @@ export const SquidList = () => {
   const [pageNumber, setPageNumber] = useState(1);
   const [pageCount, setPageCount] = useState(1);
 
-  const { data = {}, isLoading } = useSquidList({ pageIndex: pageNumber - 1, itemsPerPage });
-
-  const squidData = data.squids || [];
+  const {
+    data,
+    isLoading,
+    refetch: refetchSquids,
+  } = useSquidList({ pageIndex: pageNumber - 1, itemsPerPage });
 
   useEffect(() => {
-    setPageCount(data.pageCount);
+    setPageCount(data?.pageCount);
   }, [data]);
 
-  const squids = squidData.map((squid) => <SquidInfo key={squid.id} {...squid} />);
-
   const handlePageClick = (event) => {
     setPageNumber(event.selected + 1);
   };
 
-  let pageHeader;
+  let pageHeader = <p className="squidList__heading">(page 1 / 1)</p>;
```

Comment:
I like the default value here -- one suggestion is to set default values for pageNumber and pageCount and only assign pageHeader once -- this way if you ever wanted to change the text (e.g., to `page 1 of 1`), you only have to change it in one place

Reply:
Totally - I had this because on initial render for a split second it showed `undefined` for the `pageCount`, but cleaned this up to one line!